### PR TITLE
feat: revert tauri  cli package to beta[]

### PR DESCRIPTION
**Pull request by LordNayan**
## Description

Revert tauri cli to beta version to fix build break on both windows and mac.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [x] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: apps/@sparrow-desktop/src-tauri/Cargo.toml [hunk 1]
```diff
@@ -50,10 +50,3 @@ tauri-plugin-fs = "2.2.0"
 tauri-plugin-deep-link = "2.2.0"
 [target.'cfg(target_os = "macos")'.dependencies]
 objc = "0.2"
-
-[features]
```
Comment (LordNayan) [inline apps/@sparrow-desktop/src-tauri/Cargo.toml:54]: Not required - See breaking change 

https://tauri.app/release/@tauri-apps/cli/v2.0.0-beta.5/

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**